support/get-resources.sh

```diff
@@ -58,7 +58,7 @@ cat << EOF > ${work_file}
     },
     {
       "id": "apps",
-      "cmd": "oc get $(oc api-resources --api-group=apps --no-headers | awk '{print $1}' | paste -d, -s) --all-namespaces -o json",
+      "cmd": "oc get $(echo $(oc api-resources --api-group=apps --no-headers | awk '{print $1}') | sed 's/ /,/g' | awk '{print $1",replicationcontrollers"}') --all-namespaces -o json",
```

Comment:
I believe they should be included here. `ReplicationControllers` are the owners of pods created for `DeploymentConfigs`, the same as `ReplicaSets` for pods created for `Deployments` or `StatefulSets`.  `DeploymentConfigs` are an OpenShift, not k8s concept, so I am assuming they started to use `ReplicationControllers` before `ReplicaSets` existed but both of these resource types essential are identical.

Reply:
After looking a little more at the structure of the resource types in the apps api group, I think the reason for ReplicationControllers not being part of the group is because they don't support ControllerRevisions like the others in app group.  What they do is so similar that I can see how including them in the same oc get would make sense, but ideally, the query would be named something other than apps.  Unfortunately, that would impact backward compatibility for nothing more than a label change so happy to keep this the way you have.